src/shippingservice/main.go

```diff
@@ -128,30 +147,22 @@ func (s *server) ShipOrder(ctx context.Context, in *pb.ShipOrderRequest) (*pb.Sh
 	}, nil
 }
 
-func initStats(exporter *stackdriver.Exporter) {
-	view.SetReportingPeriod(60 * time.Second)
-	view.RegisterExporter(exporter)
-	if err := view.Register(ocgrpc.DefaultServerViews...); err != nil {
-		log.Warn("Error registering default server views")
-	} else {
-		log.Info("Registered default server views")
-	}
-}
-
-func initStackDriverTracing() {
-	// TODO(ahmetb) this method is duplicated in other microservices using Go
-	// since they are not sharing packages.
+// Initialize Stats using OpenCensus
+// TODO: remove this after conversion to OpenTelemetry Metrics
+func initOpenCensusStats() {
```

Comment:
This function seems very similar to a function in the checkout service. Is there any way to simplify this to one function across the services (if it is the case that they're similar enough)?

Reply:
they're identical, yes :) the problem is these services are isolated and don't share packages, which is intentional. it's possible to reuse, but it's a small amount of code and we're planning on deleting it anyway